Documentation/gettingstarted_kafka.rst

```diff
@@ -0,0 +1,410 @@
+agent and the Cilium CNI plugin.
+
+To deploy Cilium, run:
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes/cilium.yaml
+    clusterrole "cilium" created
+    serviceaccount "cilium" created
+    clusterrolebinding "cilium" created
+    configmap "cilium-config" created
+    secret "cilium-etcd-secrets" created
+    daemonset "cilium" created
+
+Kubernetes is now deploying Cilium with its RBAC, ConfigMap and Daemon Set as a
+pod on all cluster nodes. This operation is performed in the background.
+
+Run the following command to check the progress of the deployment:
+
+::
+
+    $ kubectl get ds --namespace kube-system
+    NAME            DESIRED   CURRENT   READY     NODE-SELECTOR   AGE
+    cilium          1         1         0         <none>          2m
+
+Wait until the cilium Deployment shows a ``CURRENT`` count of ``1`` like above (a READY value of 0 is OK
+for this tutorial).
+
+Step 2: Deploy the Demo Application
+===================================
+
+Now that we have Cilium deployed and ``kube-dns`` operating correctly we can
+deploy our demo Kafka application.  Since our first demo of Cilium + HTTP-aware security
+policies was Star Wars themed we decided to do the same for Kafka.  While the
+`HTTP-aware Cilium  Star Wars demo <https://www.cilium.io/blog/2017/5/4/demo-may-the-force-be-with-you>`_
+showed how the Galactic Empire used HTTP-aware security policies to protect the Death Star from the
+Rebel Alliance, this Kafka demo shows how the lack of Kafka-aware security policies allowed the
+Rebels to steal the Death Star plans in the first place.
+
+Kafka is a powerful platform for passing datastreams between different components of an application.
+A cluster of "Kafka brokers" connect nodes that "produce" data into a data stream, or "consume" data
+from a datastream.   Kafka refers to each datastream as a "topic".
+Because scalable and highly-available Kafka clusters are non-trivial to run, the same cluster of
+Kafka brokers to handle many different topics at once (read this `Introduction to Kafka
+<https://kafka.apache.org/intro>`_ for more background).
+
+In our simple example, the Empire uses a Kafka cluster to handle two different topics:
+
+- *empire-announce* : Used to broadcast announcements to sites spread across the galaxy
+- *deathstar-plans* : Used by a small group of sites coordinating on building the ultimate battlestation.
+
+To keep the setup small, we will just launch a small number of pods to represent this setup:
+
+- *kafka-broker* : A single pod running Kafka and Zookeeper representing the Kafka cluster
+  (labels app=kafka)
+- *empire-hq* : A pod representing the Empire's Headquarters, which is the only pod that should
+  produce messages to *empire-announce* or *deathstar-plans* (labels app=empire-hq)
+- *empire-backup* : A secure backup facility, which is allowed to "consume" from the secret
```

Comment:
Needs to be renamed ;-)

The Scarif vault was a six-story-high data vault located within the Citadel Tower of the Imperial security complex on the tropical Outer Rim planet of Scarif. The vault held scientific treatises, bureaucratic memoranda, and schematics, including the plans to the Galactic Empire's planet-destroying battle station, the Death Star.

http://starwars.wikia.com/wiki/Scarif_vault

Reply:
hahaha.... I had a vague memory from Rogue one that there was a specific place I should name this to.